datasets/titanic/metadata.json

```diff
@@ -32,11 +45,11 @@
         "key": "#{key}",
         "field": [
             { "name": "key", "@type": "ml:Field", "dataType": "sc:Integer" },
-            { "name": "label", "@type": "ml:Field", "dataType": "sc:String" },
+            { "name": "label", "@type": "ml:Field", "dataType": "sc:Text" },
             {
                 "name": "url",
                 "@type": "ml:Field",
-                "dataType": ["sc:Url", "wd:48277"]
+                "dataType": ["sc:url", "wd:48277"]
```

Comment:
sc:url is a property, not a type. I think you want sc:URL here.

We discussed with @pierrot0  whether we want to allow properties as well, but I don't think we reached a definitive conclusion on that yet.

Reply:
What should the verifier check for as far as dataTypes are concerned? How can I verify that a URI defines a dataType?

Reply:
I think for schema.org and other base types we want to support , it's reasonable to check that the type is one that we know about. It could generate a warning in case of an unknown type.